UnitTests/GitUITests/CommandsDialogs/CommitDialog/FormCommitTests.cs

```diff
@@ -85,42 +86,57 @@ public void Should_update_committer_info_on_form_activated()
         [Test]
         public void Should_display_branch_and_no_remote_info_in_statusbar()
         {
-            _referenceRepository.CheckoutMaster();
-            RunFormTest(form =>
+            var repository = new ReferenceRepository(new GitModuleTestHelper("branch_and_no_remote"));
```

Comment:
NB: `repository` will need to be disposed of so the underlying folder is deleted

Reply:
Thanks. done

Reply:
@RussKie can you cancel the current build that is stuck and also cancel the builds that  are not related to building code?

Reply:
done
you can reach me faster on gitter